dss/events/handlers/index.py

```diff
@@ -76,44 +106,45 @@ def create_index_data(s3, bucket_name, manifest, logger):
             # of the query spec, so go with that, at least for now. Do so by substituting
             # dot for underscore in the key filename portion of the index.
             # As due diligence, additional investigation should be performed.
-            index_filename = file_info["name"].replace(".", "_")
+            index_filename = file_info[BundleFileMetadata.NAME].replace(".", "_")
```

Comment:
By aliasing issue, are you referring to index document keys representing file names have an underscore in place of a dot? From what I've read, dot in ES doc keys can be problematic, and are best avoided. The metadata group is independently avoiding dot in key names. This does mean that there is not a simple, reliable reverse translation from the key names in the index document to the actual file names, which I agree is an issue. No, I don't have a plan for how to address that issue.